docs/tutorials/03_authenticated_user.md

```diff
@@ -0,0 +1,98 @@
+This tutorial assumes you have already followed:
+
+- [Hello World Tutorial]{@tutorial 01_hello_world}
+- [Ephemeral User Tutorial]{@tutorial 02_ephemeral_user}
+
+Complete code for this tutorial is available at: {@link https://github.com/ForstaLabs/forsta-messenger-client/tree/master/examples/authenticated_user}
+
+In the last tutorial we configured the client to use an ephemeral user who is
+created and logged in based on the org ephemeral token. However in production
+systems we recommend creating Forsta users using the Forsta Atlas API and 
+libraries. In this tutorial we will discuss how to connect an authenticated user.
+
+You can create a user using either:
+
+- [Forsta Console]{@link https://app.forsta.io/people}
+- [Atlas API]{@link https://atlas.forsta.io/doc/}
+
+For a user to authenticate with the Forsta Messenger Client they need to provide
+a Forsta generated JWT (JSON Web Token). In a production system JWTs are generated
+by authenticating a user with a saved auth token. For testing purposes you can
+also generate a JWT with the following shell scripts:
+
+- [JWT Generator]{@link https://github.com/ForstaLabs/developer-examples/blob/master/scripts/get_jwt.sh}
+- [Required Forsta Script Library]{@link https://github.com/ForstaLabs/developer-examples/blob/master/scripts/forsta_library.sh}
```

Comment:
I'm confused by these entries.  What does "Required" mean?  E.g. Do I have to get that shell script and run it??  It seems to take precedence over the JWT Generator entry with that strong language and I'm not sure what I'm even looking at.  If those shells scripts are doing some API call, I would almost prefer that we just suggest which API endpoint they are going to and point do our openapi based refs and only then, offer a shell snippet (preferably inline).  Anytime a doc asks me to run shell scripts blind, I get nervous (and usually don't comply).

Reply:
Good point. How about we point them at the correct API end points and provide the directory with the shell scripts as example code showing how the APIs are called. Long term this will be replaced by references to our assorted Atlas libraries in different languages.. but we don't have those yet.

Reply:
I'm adding an external definition for JWT.